.github/workflows/nightly-docker.yml

```diff
@@ -0,0 +1,84 @@
+name: "Nightly Docker Image Build"
+
+on:
+  schedule:
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  build-phar:
+    permissions:
+      contents: read
+      id-token: write
+      attestations: write
+    uses: ./.github/workflows/build-phar.yml
+
+  docker-nightly-image:
+    needs: build-phar
+    name: Docker nightly image
+    runs-on: ubuntu-latest
+
+    permissions:
+      attestations: write
+      id-token: write
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v5
+
+      - name: Fetch built PHAR from artifacts
+        uses: actions/download-artifact@v5
+        with:
+          name: pie-${{ github.sha }}.phar
+
+      - name: Verify the PHAR
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: gh attestation verify pie.phar --repo ${{ github.repository }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          flavor: |
+            latest=false
+          images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=raw,value=nightly
```

Comment:
```suggestion
```

Reply:
Ah yes, I thought we wanted both tags. PR updated

Reply:
Alright, thanks!